modin/api/partition_api.py

```diff
@@ -47,15 +52,13 @@ def unwrap_partitions(api_layer_object, axis=None, bind_ip=False):
         def _unwrap_partitions(oid):
             if bind_ip:
                 return [
-                    (partition.ip, getattr(partition, oid))
+                    [(partition.ip, getattr(partition, oid)) for partition in row]
```

Comment:
Because we hold all partitions as 2D np.array the `unwrap_partitions` now returns 2D list instead of 1D list when passed axis is `None`. It is that what I described at the top of the PR. This change is needed for `create_df_from_partitions` to create df from 2D list of partitions when passed axis is `None`.